Pipfile

```diff
@@ -15,6 +15,7 @@ typing-extensions = "*"
 types-requests = "*"
 typed-ast = "*"
 jinja2 = "~=3.0.1"
+browser_cookie3 = "*"
```

Comment:
Could you rebase this PR to `upcoming/v4.11` and then do `pipenv install browser_cookie3` to properly update the Pipfile and Pipfile.lock?

Reply:
Will do once all the PR reviews are approved.

Reply:
Looks good, I think this is ready to be rebased :)

Reply:
@aandergr Rebased is completed, Pipfile and Pipfile.lock are updated as well.
Let me know if everything looks right :)